BroFrame.cpp

```diff
@@ -946,6 +946,40 @@ void CBrowserFrame::CreateRebars()
 			}
 		}
 	}
+	else
+	{
+		// Even though IsRebar option is disabled (EnableRebar=0), create rebar as a
+		// container for only packing cTabWnd purpose.
+		// It fixes the issue (When rebar is disabled, the top of website content 
+		// in tab is hidden unexpectedly, in other words,
+		// the content rendering is clipped and can't access hidden parts)
+		if (m_cTabWnd)
+		{
+			if (m_pwndReBar == NULL)

```

Comment:
Can we remove this `if` check because `m_pwndReBar` seems be always NULL at this line?

Reply:
Thanks, we can remove it.